mm2src/mm2_main/src/lp_native_dex.rs

```diff
@@ -534,6 +538,13 @@ fn p2p_precheck(ctx: &MmArc) -> P2PResult<()> {
         }
     }
 
+    if is_seed_node {
+        let crypto_initialized = CryptoCtx::is_init(ctx)?;
```

Comment:
Can this ever fail? If it's not infallible, shouldn't we consider failures as false, like `unwrap_or(false)`?